java/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.java

```diff
@@ -78,9 +81,13 @@
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
-    setContentView(R.layout.activity_checkout);
     initializeUi();
 
+    // Create notification channels according to Android O+ guidelines
```

Comment:
I would prefer if notification channels are created outside of the activity, but for a sample it might be ok. If you have a custom Application class, I think it would fit better there

Reply:
Yep, same feeling. The intention is to avoid that added complexity.
A good next step when the application grows.